framework/SupervisedLearning/__init__.py

```diff
@@ -45,5 +45,6 @@
 from .KerasMLPClassifier import KerasMLPClassifier
 from .KerasConvNetClassifier import KerasConvNetClassifier
 from .KerasLSTMClassifier import KerasLSTMClassifier
+from .KerasLSTMRegression import KerasLSTMRegression
```

Comment:
These imports are not needed based on @alfoa comments

Reply:
Removed KerasLSTMClassifier and KerasLSTMRegression